### PR TITLE
SCRUM-1840 load ATP ontology

### PR DESCRIPTION
**Pull request by cmpich**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java [hunk 1]
```diff
@@ -82,115 +116,142 @@ public void process(BulkLoadFile bulkLoadFile) throws Exception {
...
-				processTerms(bulkLoadFile,xpoTermService, config);
-			} else if(ontologyType == OntologyBulkLoadType.XBED) {
-				processTerms(bulkLoadFile, xbedTermService, config);
-			} else if(ontologyType == OntologyBulkLoadType.XSMO) {
-				processTerms(bulkLoadFile, xsmoTermService, config);
-			} else {
-				log.info("Ontology Load: " + bulkLoadFile.getBulkLoad().getName() + " for OT: " + ontologyType + " not implemented");
-				throw new Exception("Ontolgy Load: " + bulkLoadFile.getBulkLoad().getName() + " for OT: " + ontologyType + " not implemented");
+
+			switch (ontologyType) {
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java:151]: @cmpich thanks... this looks much better.

File: src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java [hunk 2]
```diff
@@ -82,115 +116,142 @@ public void process(BulkLoadFile bulkLoadFile) throws Exception {
...
+					GenericOntologyLoadConfig config2 = new GenericOntologyLoadConfig();
+					config2.getAltNameSpaces().add("xenopus_developmental_stage");
+					processTerms(bulkLoadFile, OntologyBulkLoadType.XBS, xbsTermService, config2);
+					break;
+				case XPO:
+					config.setLoadOnlyIRIPrefix("XPO");
+					processTerms(bulkLoadFile, xpoTermService, config);
+					break;
+				case ATP:
+					processTerms(bulkLoadFile, atpTermService, config);
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java:231]: @cmpich you will need to do a `config.setLoadOnlyIRIPrefix("ATP");` here also.

**Review by oblodgett — CHANGES_REQUESTED** [review]

File: src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java (hunk 2 above)
Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java:231]: @cmpich the config needs to happen before the process happens.

Comment (cmpich) [reply src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadJobExecutor.java:231]: Sure.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**